pythonosc/osc_server.py

```diff
@@ -171,11 +171,19 @@ def datagram_received(self, data, unused_addr):
       _call_handlers_for_packet(data, self.dispatcher)
 
   def serve(self):
-    """creates a datagram endpoint and registers it with our event loop"""
+    """creates a datagram endpoint and registers it with our event loop.
+
+    Use this only if your are not currently running your asyncio loop.
```

Comment:
Should be "only if _you're_ not".  "Your" is the possessive.

Also, you can't capitalize the "i" in "i.e.", because it cannot grammatically start a sentence.  :-)  Consider replacing the . on line 176 with a comma!